### PR TITLE
Make the release automatic

### PR DESCRIPTION
**Pull request by konifar**
## Issue
- ref #439

## Overview (Required)
- Modified play-publisher settings
- Added metadata files

### CODE REVIEW

**Comment by konifar** [comment]
This has been almost done but publisher-keys.json doesn't work well because of the authorization issue 🙇  I'll ask to the member who is the owner account.

**Comment by konifar** [comment]
Finally, it works!

**Comment by takahirom** [comment]
🎉

File: README.md [hunk 1]
```diff
@@ -251,6 +251,19 @@ SessionsFragment.kt
         })
 ```
 
+## Release
```
Comment (takahirom) [inline README.md]: 👍

File: app/src/main/play/ja-JP/listing/fulldescription [hunk 2]
```diff
@@ -0,0 +1,7 @@
+2018年2月8・9日に開催予定の DroidKaigi 2018 の公式アプリです。セッションのスケジュール確認や当日のお供にお使いください。
```
Comment (takahirom) [inline app/src/main/play/ja-JP/listing/fulldescription]: Will English descriptions are deleted? 🤔

Comment (konifar) [reply app/src/main/play/ja-JP/listing/fulldescription]: Ah, you have already added it?
I'll add `en/listing`.

Comment (takahirom) [reply app/src/main/play/ja-JP/listing/fulldescription]: Thanks 👍

Comment (takahirom) [reply app/src/main/play/ja-JP/listing/fulldescription]: I already added it 😉

Comment (konifar) [reply app/src/main/play/ja-JP/listing/fulldescription]: 👍  But I can't find English `whatsnew` in Google Play console.
Can you create it later?

Comment (takahirom) [reply app/src/main/play/ja-JP/listing/fulldescription]: Well.. 


```
ついにDroidKaigiも今週になりましたね！
見に行くセッションは決まってきましたか？
公式アプリはまだまだコントリビューターのみなさんの修正を受け付けております！
開始時間ごとにセッションが見やすくなりました。
お知らせを通知するようになりました。
```

Google translated..

```
At last DroidKaigi got this week!
Have you decided on sessions to go see?
The official application is still accepting contributors' contributions!
The session became easier to see at each start time.
I now notify you of notifications.
```

Comment (panpanini) [reply app/src/main/play/ja-JP/listing/fulldescription]: :eyes: please wait a moment

Comment (panpanini) [reply app/src/main/play/ja-JP/listing/fulldescription]: お待たせしました！
```
DroidKaigi is finally upon us!
Have you decided which sessions you want to see?
The official app is still accepting contributions!

Changes:
Its now easier to see when each session starts
Information will now be sent as push notifications
```
でどうでしょうか！

Comment (takahirom) [reply app/src/main/play/ja-JP/listing/fulldescription]: Thank you very much 🔥

Comment (konifar) [reply app/src/main/play/ja-JP/listing/fulldescription]: Thanks! I added English metadata.
Can you review again?

**Comment by takahirom** [comment]
Thanks Let's merge this🚀

**Pull request merged**